Assets/PatchKit Patcher/Scripts/Patcher.cs

```diff
@@ -916,6 +927,80 @@ private void ThreadUpdateApp(bool automatically, CancellationToken cancellationT
                 }
             }
         }
+        
+        private void ThreadVerifyAllAppFiles(CancellationToken cancellationToken)
+        {
+            _state.Value = PatcherState.UpdatingApp;
```

Comment:
Please introduce here a `// TODO:` comment as we might want to introduce here a new state.

Reply:
done